.test-defs/CreateShoot.yaml

```diff
@@ -42,8 +42,8 @@ spec:
     -kube-apiserver-min-allowed-memory=$KUBE_APISERVER_MIN_ALLOWED_MEMORY
     -etcd-min-allowed-cpu=$ETCD_MIN_ALLOWED_CPU
     -etcd-min-allowed-memory=$ETCD_MIN_ALLOWED_MEMORY
+    -machine-image-version=$MACHINE_IMAGE_VERSION
```

Comment:
/cc @hendrikKahl
This will allow us to set the machine image versions for worker pools during shoot creation.

Reply:
Shouldn't `-machine-image-version` come after `-machine-image-name`? I prefer the previous order

Reply:
Or we can uncomment that line too